requirements/dev.txt

```diff
@@ -136,22 +275,110 @@ cwcwidth==0.1.6 \
+    --hash=sha256:233dcde5d6895416f5613e4ca030e88f18a163e4b676e7aa815d191754f53085 \
+    --hash=sha256:d41db694bbf3085b026df227428427b859478a2846b55088429e3ccc7115c0c5
+    # via -r requirements/prod.txt
+django-jsonview==2.0.0 \
+    --hash=sha256:1871983f6c7ab0bec1bad3249b06ce64c47d3bed57cd190f87e5acaf65722ebb \
+    --hash=sha256:77345393f8e8bb3279b9ba9fb2db687adfa74d41feab972521545d47103d321b
+    # via -r requirements/prod.txt
+django-memoize==2.3.1 \
+    --hash=sha256:62ac4807710ecf22a7397d008d64d0f798e7230482d4b6072d916cac852a47cd
+    # via -r requirements/prod.txt
+django-mozilla-product-details==1.0.1 \
+    --hash=sha256:484b61deb99663c131c7e386bc683c83cab612777055b12588e91844a1508a04 \
+    --hash=sha256:ef25e5e9bce4bfe6c7ba9c87245f5be93c0ac3fc7f1d60afd459e55e9dcc8bc7
+    # via -r requirements/prod.txt
+django-watchman==1.2.0 \
+    --hash=sha256:6c0b8889456ed644fcfe2f7c3294cb4ef8568a85772b7f95842e2d8c9b4bbff5 \
+    --hash=sha256:c38830c58984b8eb29db30a3e332968d1c7e235dee3f5c0a907d8d79a37a3125
+    # via -r requirements/prod.txt
+docutils==0.17.1 \
+    --hash=sha256:686577d2e4c32380bb50cbb22f575ed742d58168cee37e99117a854bcd88f125 \
+    --hash=sha256:cf316c8370a737a022b72b56874f6602acf974a37a9fba42ec2876387549fc61
+    # via -r requirements/prod.txt
+envcat==0.1.1 \
+    --hash=sha256:3659c6bad8f47cd3c2691d754a5659c1953480c066718c8a1d87367b03f11bc8 \
+    --hash=sha256:6831806d205b7b1c8456bba5d9cd7ced90f52cf2fc8cfbe234d48d3fdb91111d
+    # via -r requirements/prod.txt
+everett==3.0.0 \
+    --hash=sha256:43c208e09ef53adb8e8739ad96999893a2c0a27753e33f89be346e916e71efb0 \
+    --hash=sha256:80ce592168ce50643711eceff3ec94fdeec755f228de746f20b1bfc94ea20682
+    # via -r requirements/prod.txt
 factory-boy==3.2.1 \
     --hash=sha256:a98d277b0c047c75eb6e4ab8508a7f81fb03d2cb21986f627913546ef7a2a55e \
     --hash=sha256:eb02a7dd1b577ef606b75a253b9818e6f9eaf996d94449c9d5ebb124f90dc795
-    # via -r requirements/inputs/dev.in
+    # via -r requirements/dev.in
 faker==13.2.0 \
     --hash=sha256:206050b40d6dfb6efe8ccfbbb9866c56477c53edfdab83f80de3b59f38a88ce0 \
     --hash=sha256:edb62f9511a0977abcea6a78e43b99d1f26c915d2142bbf4c6bf02a9bb597c7f
     # via factory-boy
 flake8==4.0.1 \
     --hash=sha256:479b1304f72536a55948cb40a32dce8bb0ffe3501e26eaf292c7e60eb5e0428d \
     --hash=sha256:806e034dda44114815e23c16ef92f95c91e4c71100ff52813adf7132a6ad870d
-    # via -r requirements/inputs/dev.in
+    # via -r requirements/dev.in
+fluent-runtime==0.3 \
+    --hash=sha256:8d41d4bf4cb19039370e14de78f2991cae9721d44e66165989a69ff5c22d90f0 \
+    --hash=sha256:d8a81a36d111ba3e66af27f9224e94ef305cf3bec407c3f98a7c597eef554652
+    # via -r requirements/prod.txt
 fluent-syntax==0.17.0 \
     --hash=sha256:ac3db2f77d62b032fdf1f17ef5c390b7801a9e9fb58d41eca3825c0d47b88d79 \
     --hash=sha256:e26be470aeebe4badd84f7bb0b648414e0f2ef95d26e5336d634af99e402ea61
-    # via compare-locales
+    # via -r requirements/prod.txt
+fluent-syntax==0.17.0 \
```

Comment:
This is a duplicate, but can't get rid of it. However, it won't cause installation problems, just a little wasted time/CPU